simulator.py

```diff
@@ -167,7 +167,7 @@ def is_infectious(self):
 
     @property
     def is_removed(self):
-        return self.recovered_timestamp == datetime.datetime.max
+        return self.recovered_timestamp != datetime.datetime.min
```

Comment:
when i coded this, i was thinking -

- in the beginning: assign `datetime.datetime.min` to `self.recovered_timestamp`
- (reinfection possible) if someone is infected and has recovered assign `self.env.timestamp` to `self.recovered_timestamp`. This `timestamp` will be used to get time delta between two infections to the same person. 
-  (reinfection possible) if someone is dead assign `datetime.datetime.max` to `self.recovered_timestamp`

So `is_removed` was `self.recovered_timestamp == datetime.datetime.max` (or the agent never recovers).  

The current clause will make the reinfected susceptible agents removed from the population.